omero/sysadmins/version-requirements.rst

```diff
@@ -62,15 +65,19 @@ Operating systems
      * - Linux version
        - OMERO 5.3
        - OMERO 5.4
+       - OMERO 5.5
      * - 12.04
        - |Dropped|
        - |Dropped|
+       - |Dropped|
```

Comment:
Since nothing is supported from `12.04`, should we unlist it?

Reply:
good idea